sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -24,6 +24,19 @@ const AzureAccountClientId = "aebc6443-996d-45c2-90f0-388ff96faa56"; // VSC: 'ae
 const VSCodeUserName = "VS Code Azure";
 const logger = credentialLogger("VisualStudioCodeCredential");
 
+// Map of unsupported Tenant IDs and the errors we will be throwing.
+const unsupportedTenantIds: Record<string, string> = {
+  adfs: "The VisualStudioCodeCredential is not supported against ADFS tenant / authorities."
```

Comment:
Slight rewording, @schaabs may need to check for accuracy:

```suggestion
  adfs: "The VisualStudioCodeCredential does not support authentication with ADFS tenants."
```

Reply:
Perfect! Thank you! 🌞